apps/front/src/styles/_ratio.scss

```diff
@@ -119,3 +121,27 @@
     }
   }
 }
+
+/// Desktop media query.
+/// @param {string} [$breakpoint=breakpoints.$breakpoint-tablet] - Le point de rupture pour les médias queries (par défaut égal à breakpoints.$breakpoint-tablet).
+@mixin desktop($breakpoint: breakpoints.$breakpoint-tablet) {
+  @media (min-width: $breakpoint) and (orientation: landscape) and (min-height: breakpoints.$breakpoint-mobile-horizontal) {
```

Comment:
update order @media (min-width: $breakpoint)  and (min-height: breakpoints.$breakpoint-mobile-horizontal) and (orientation: landscape)